packages/fxa-settings/src/models/Account.ts

```diff
@@ -538,7 +540,10 @@ export class Account implements AccountData {
     });
   }
 
-  async resetPassword(email: string): Promise<PasswordForgotSendCodePayload> {
+  async resetPassword(
+    email: string,
+    service?: string
```

Comment:
I'm not sure if this would have been preferred anyway but I tried using `CreateRelier` here, and ran into issues. There was nothing in the console, so I worked my way up and saw this was struggling to pull off of `Context` here, maybe because this isn't a React component? Didn't dive too deep, since we're replacing links with codes soon anyway, and just called `CreateRelier` in the component using it instead.

Reply:
I think that's good. I don't think we should be accessing context here. Better to pass it in.